src/app/_components/Header/Nav/index.tsx

```diff
@@ -1,41 +1,53 @@
+///Users/hassan/yardsAndParts/src/app/_components/Header/Nav/index.tsx
 'use client'
 
 import React from 'react'
 import Link from 'next/link'
-
-import { Header as HeaderType, User } from '../../../../payload/payload-types'
 import { useAuth } from '../../../_providers/Auth'
-import { CartLink } from '../../CartLink'
-import { CMSLink } from '../../Link'
-
+import { CartLink } from '../../CartLink';
 import classes from './index.module.scss'
 
-export const HeaderNav: React.FC<{ header: HeaderType }> = ({ header }) => {
-  const navItems = header?.navItems || []
+export const HeaderNav: React.FC = () => {
   const { user } = useAuth()
 
   return (
-    <nav
-      className={[
-        classes.nav,
-        // fade the nav in on user load to avoid flash of content and layout shift
-        // Vercel also does this in their own website header, see https://vercel.com
-        user === undefined && classes.hide,
-      ]
-        .filter(Boolean)
-        .join(' ')}
-    >
-      {navItems.map(({ link }, i) => {
-        return <CMSLink key={i} {...link} appearance="none" />
-      })}
-      <CartLink />
-      {user && <Link href="/account">Account</Link>}
-      {!user && (
-        <React.Fragment>
-          <Link href="/login">Login</Link>
-          <Link href="/create-account">Create Account</Link>
-        </React.Fragment>
-      )}
-    </nav>
+    <>
+      <nav className={classes.nav}>
+        <div className={classes.left}>
+          <button className={classes.categoriesButton}>
+            <span className={classes.icon}>☰</span>
+            All categories
+            <span className={classes.arrow}>▼</span>
+          </button>
+        </div>
+        <div className={classes.center}>
+          <Link href="/" className={classes.navLink}>Home</Link>
```

Comment:
can you please follow the below code instead of hard coding, you can check out the video how to use it.

{navItems.map(({ link }, i) => {
        return <CMSLink key={i} {...link} appearance="none" />
      })}